modules/katello_activation_key.py

```diff
@@ -98,137 +98,39 @@
 RETURN = '''# '''
 
 try:
-    from nailgun import entities
-    from nailgun.config import ServerConfig
-    from ansible.module_utils.ansible_nailgun_cement import fields_equal
+    from ansible.module_utils.ansible_nailgun_cement import (
+        create_server,
+        ping_server,
+        fields_equal,
```

Comment:
I don't think this is used

Reply:
correct, that was a stray copy&paste from the old code, where it was not used